.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
 node_js:
-- 6.0
+- 7
 notifications:
   email: false
 sudo: false
 before_install:
-  openssl aes-256-cbc -K $encrypted_ca4a154c2e6b_key -iv $encrypted_ca4a154c2e6b_iv -in config/tenon.json.enc -out config/tenon.json -d
+  - if [[ `npm -v` != 5* ]]; then npm i -g npm@5.0.3; fi
```

Comment:
Should we try `npm i -g npm@latest`? If this does eventually end up biting us we can lock to a particular version.